r2i/onnxrt_openvino/parameters.h

```diff
@@ -0,0 +1,161 @@
+/* Copyright (C) 2020 RidgeRun, LLC (http://www.ridgerun.com)
+ * All Rights Reserved.
+ *
+ * The contents of this software are proprietary and confidential to RidgeRun,
+ * LLC.  No part of this program may be photocopied, reproduced or translated
+ * into another programming language without prior written consent of
+ * RidgeRun, LLC.  The user is free to modify the source code after obtaining
+ * a software license from RidgeRun.  All source code changes must be provided
+ * back to RidgeRun without any encumbrance.
+*/
+
+#ifndef R2I_ONNXRT_OPENVINO_PARAMETERS_H
+#define R2I_ONNXRT_OPENVINO_PARAMETERS_H
+
+#include <memory>
+#include <string>
+#include <unordered_map>
+
+#include <r2i/iparameters.h>
+#include <r2i/runtimeerror.h>
+#include <r2i/onnxrt_openvino/engine.h>
+#include <r2i/onnxrt/parameters.h>
+
+namespace r2i {
+namespace onnxrt_openvino {
+
+class Parameters: public r2i::onnxrt::Parameters {
+ public:
+  Parameters ();
+  RuntimeError Configure (std::shared_ptr<IEngine> in_engine,
+                          std::shared_ptr<IModel> in_model) override;
+  std::shared_ptr<IEngine> GetEngine () override;
+  RuntimeError Get (const std::string &in_parameter, int &value) override;
+  RuntimeError Get (const std::string &in_parameter, double &value) override;
+  RuntimeError Get (const std::string &in_parameter,
+                    std::string &value) override;
+  RuntimeError Set (const std::string &in_parameter,
+                    const std::string &in_value) override;
+  RuntimeError Set (const std::string &in_parameter, int in_value) override;
+  RuntimeError Set (const std::string &in_parameter, double in_value) override;
+  RuntimeError List (std::vector<ParameterMeta> &metas) override;
+
+ private:
+  std::shared_ptr <Engine> engine;
+
+  friend class Accessor;
+
+  class Accessor {
+   public:
+    Accessor (Parameters *target) : target(target) {}
+    virtual RuntimeError Set () = 0;
+    virtual RuntimeError Get () = 0;
+    virtual ~Accessor() {}
+
+   public:
```

Comment:
Why do we need public twice in this class?

Reply:
We don't. I just noticed this is in all the other backends aswell. It doesn't affect anything but look ugly.